src/test/haxe/yloader/util/ParameterUtilTest.hx

```diff
@@ -98,4 +98,35 @@ class ParameterUtilTest
 		Assert.areEqual("foo", ParameterUtil.fromQueryString("?foo=1&foo=2&foo=3")[2].name);
 		Assert.areEqual("3", ParameterUtil.fromQueryString("?foo=1&foo=2&foo=3")[2].value);
 	}
+
+	@Test
+	public function customHeader_fromText_matches():Void
+	{
+		var header = "hello:world\nfoo : bar";
+		var params = ParameterUtil.fromText(header);
+
+		Assert.areEqual(2, params.length);
+		Assert.areEqual(params[0].name, "hello");
+		Assert.areEqual(params[0].value, "world");
+		Assert.areEqual(params[1].name, "foo");
+		Assert.areEqual(params[1].value, "bar");
+	}
+
+	@Test
+	public function getContentLength_matches():Void
```

Comment:
format of unit test function name is INPUTDATA_METHOD_EXPECTATION... I would call this customHeaders_getContentLength_matches